frontend/src/modules/Dashboard/Components/GroupCard.tsx

```diff
@@ -37,20 +39,54 @@ export const GroupCard = ({
     }
     setOpen(false)
   }
+  // returns color of background, button, and if newly matchable
+  function getColor(students: number, groups: number) {
+    //all students are matched
+    if (students === 0 && groups > 0) {
+      return [colors.paleviolet, colors.darkpurple, 'no']
+    }
+    //students are ready to be matched
+    else if (newStudents > 0 && groupsFormed > 0) {
+      return [colors.lightgreen, colors.darkgreen, 'no']
+    }
+    //NEWLY MATCHABLE
+    else if (newStudents > 1 && groupsFormed === 0) {
+      return [colors.lightgreen, colors.darkgreen, 'yes']
+    }
+    //only 1 student & 0 groups formed
+    else return [colors.lightyellow, colors.darkyellow, 'no']
+  }
+  const styleArray = getColor(newStudents, groupsFormed)
 
   return (
-    <StyledContainer key={key}>
+    <StyledContainer color={styleArray[0]} key={key}>
+      {styleArray[2] === 'yes' && <StyledNewlyMatchable />}
       <StyledName>{name}</StyledName>
       <StyledRows>
         <StyledRow>
-          <StyledText>{submitted} Forms Submitted</StyledText>
+          <StyledPlusIcon></StyledPlusIcon>
+          {newStudents === 1 && (
+            <StyledText>{newStudents} New Student</StyledText>
+          )}
+          {newStudents != 1 && (
+            <StyledText>{newStudents} New Students</StyledText>
+          )}
```

Comment:
I love the attention to detail, but I believe the plural can be simplified to pretty much one line, and you don't need this duplication. Only do the conditional for the s, not the whole component.